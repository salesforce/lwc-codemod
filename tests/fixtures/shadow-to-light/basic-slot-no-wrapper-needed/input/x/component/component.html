<template>
    <slot name="foo"></slot>
    <slot name="bar"></slot>
    <slot><div>hello default</div></slot>
</template>
