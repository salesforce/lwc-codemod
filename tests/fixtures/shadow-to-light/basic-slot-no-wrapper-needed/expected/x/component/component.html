<template lwc:render-mode="light">
    <slot name="foo"></slot>
    <slot name="bar"></slot>
    <slot><div>hello default</div></slot>
</template>
